### ISSUE TITLE
WPT tests runner Python error

### ISSUE THREAD

**Issue by gterzian** [servo/servo#18708]
When I run `./mach test-wpt tests/wpt/web-platform-tests/fetch/http-cache`

I get 
```
File "/servo/tests/wpt/grouping_formatter.py", line 108, in suite_start
    self.number_of_tests = sum(len(tests) for tests in data["tests"].itervalues())
AttributeError: 'list' object has no attribute 'itervalues'
```

removing the call to `itervalues` at https://github.com/servo/servo/blob/e13f5a656af682add10cdc41fb8e625c04b60347/tests/wpt/grouping_formatter.py#L108 fixed it for me locally. Seems like the data is now a list, and not a dict?

Is this worth a PR or I am just getting something wrong in my local setup?

**Comment by jdm** [comment]
It is definitely worth a PR.

**Comment by KiChjang** [comment]
Fixed in #18724.
